lib/membrane_http_adaptive_stream/bandwidth_calculator.ex

```diff
@@ -43,9 +44,14 @@ defmodule Membrane.HTTPAdaptiveStream.BandwidthCalculator do
       validated_segments_meta |> Enum.map(fn {bits_size, duration} -> bits_size / duration end)
 
     cond do
-      segments_meta |> Enum.empty?() -> @default_bandwidth
-      validated_segments_meta |> Enum.empty?() -> segments_bitrates |> Enum.max() |> trunc()
-      true -> validated_segments_bitrates |> Enum.max() |> trunc()
+      segments_meta |> Enum.empty?() ->
```

Comment:
nit: IMO if calls are not chained it's better to avoid using `|>`.
```suggestion
       Enum.empty?(segments_meta) ->
```